### PR TITLE
Add Python Django to the-scan-project

### PR DESCRIPTION
**Pull request by adamr88**
References https://github.com/the-scan-project/tsp-vulnerable-app-base/pull/1

### CODE REVIEW

File: proj/settings.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-$^e#o&rg#c$)114)g!mgn=b_#$8n5hsma2r7xoaf-%-0o^ei4g'
```
Comment (adamr88) [inline proj/settings.py]: GitGuardian is going to email us about this. It *does* say "This project contains security vulnerabilities and should be only run in testing purposes." in the README.md. I defer to your judgement

File: proj/settings.py [hunk 2]
```diff
@@ -31,52 +31,53 @@
 # Application definition
 
 INSTALLED_APPS = [
-    'django.contrib.admin',
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.messages',
-    'django.contrib.staticfiles',
+    "hello.apps.HelloConfig",
```
Comment (adamr88) [inline proj/settings.py:34]: This is the only line that changed in proj/settings.py--everything else is artifact of linting

**Review by samoylenko — APPROVED** [review]
Thank you
You're the admin of this repo, I'll let you merge :)

File: proj/settings.py (hunk 1 above)
Comment (samoylenko) [reply proj/settings.py]: I'll redirect these two PRs to this repo, there's a commit offering documenting this issue as hardcoded credentials (which we also want to present, so it's very good)

(this one) https://github.com/adamr88/tsp-vulnerable-app-python-django/pull/2
(another one - FYI) https://github.com/adamr88/tsp-vulnerable-app-python-django/pull/1

**Pull request merged**